External/Plugins/CodeRefactor/Commands/Move.cs

```diff
@@ -473,37 +456,15 @@ private void MoveRefactoredFiles()
 
                     // Look for document class changes
                     // Do not use RefactoringHelper to avoid possible dialogs that we don't want
-                    Project project = (Project)PluginBase.CurrentProject;
-                    string newDocumentClass = null;
-                    string searchPattern = project.DefaultSearchFilter;
-                    foreach (string pattern in searchPattern.Split(';'))
-                    {
-                        foreach (string file in Directory.GetFiles(oldPath, pattern, SearchOption.AllDirectories))
-                        {
-                            if (project.IsDocumentClass(file))
-                            {
-                                newDocumentClass = file.Replace(oldPath, newPath);
-                                break;
-                            }
-                        }
-                        if (newDocumentClass != null) break;
-                    }
+                    string newDocumentClass;
+                    if (RefactoringHelper.TryGetDocumentClass(oldPath, out newDocumentClass)) newDocumentClass.Replace(oldPath, newPath);
```

Comment:
I've used `.Replace(oldPath, newPath)`, just because the previous was `file.Replace(oldPath, newPath)`, but I think that's not quite safe.

Shouldn't this really be 
```c#
Path.Combine(newPath, newDocumentClass.Substring(oldPath.Length))
```
Assuming (or assuring beforehand) that `oldPath` is rooted?